dataikuapi/dss/apideployer.py

```diff
@@ -166,6 +166,15 @@ def get_settings(self):
 
         return DSSAPIDeployerInfraSettings(self.client, self.infra_id, settings)
 
+    def delete(self):
+        """
+        Deletes this infra
+        You may only delete an infra if it has no deployments on it anymore.
+        """
+        return self.client._perform_empty(
```

Comment:
Does _perform_empty return anything?

Reply:
To be fair, we have a mix of return / not return in the API client on `_perform_empty`

Reply:
Nope :) I guess we can drop the return. I did it this way as it is done like pretty much everywhere else for delete endpoints but it is not a very good reason 😅

Reply:
Oh yes, I forgot python can "return" an "empty" value, nvm then

Reply:
Too late the fix is pushed 😛